diff --git a/Onboarding/vpnXXXXX-on-establish.sh b/Onboarding/vpnXXXXX-on-establish.sh
index e67e89d..51ae964 100755
--- a/Onboarding/vpnXXXXX-on-establish.sh
+++ b/Onboarding/vpnXXXXX-on-establish.sh
@@ -47,25 +47,26 @@ PEERGITREPO=/var/freifunk/peers-ffs
 DATAPATH=/var/freifunk/database
 BLACKLIST=/var/freifunk/blacklist
 
+touch $LOGFILE
 
-date >> $LOGFILE
+date 
 
-echo Starting new ffs-Onboarding Process on $INTERFACE from $PEER_ADDRESS ... >> $LOGFILE
+echo Starting new ffs-Onboarding Process on $INTERFACE from $PEER_ADDRESS ... 
 
 CL="/usr/bin/env ffs-Onboarding.py --fastd $INTERFACE --batman bat${INTERFACE:3:5} --pid $FASTD_PID --peerkey $PEER_KEY --gitrepo $PEERGITREPO --data $DATAPATH --blacklist $BLACKLIST"
 
-echo "$CL" >> $LOGFILE
+echo "$CL"
 
-eval "$CL" 2>&1 >> $LOGFILE
+eval "$CL" 2>&1
 
 if [ $? != 0 ]; then
-    date >> $LOGFILE
-    echo ++ ERROR >> $LOGFILE
+    date
+    echo ++ ERROR
     kill -s 12 $FASTD_PID    # SIGUSR2 = drop all connections
     kill -s 17 $FASTD_PID    # SIGCHLD = unlink zombies
 else
-    date >> $LOGFILE
-    echo OK. >> $LOGFILE
+    date 
+    echo OK.
 fi
 
-echo ---------------------------------------- >> $LOGFILE
+echo ----------------------------------------
